examples/zone-printer/README.md

```diff
@@ -32,227 +32,215 @@ that will be used by `kubemci`.
 
 # Tutorial
 
-## 0. Prerequisite
+## Step 0: Before you begin
 
-### 0.1 GCP Project
+1. **GCP Project**
 
-Ensure you that you have a GCP Project. If you don't have one already,
-you can create a project by following the instructions at
-[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+    Ensure you that you have a GCP Project. If you don't have one already,
+    you can create a project by following the instructions at
+    [Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
 
-After your project is created, run the following command in your
-terminal:
+    After your project is created, run the following commands in your
+    terminal:
 
-```shell
-PROJECT=<your-project-name>
-```
+    ```shell
+    PROJECT=<your-project-name>
+    ```
 
-### 0.2 `gcloud` installed and initialized
+2. **Set up `gcloud`**:
 
-You can install `gcloud` by following the instructions at
-[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
-initialize it by running the following commands:
+    You can install `gcloud` by following the instructions at
+    [Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+    initialize it by running the following commands:
 
-```shell
-gcloud init # Set $PROJECT as the default project
-gcloud auth login
-gcloud auth application-default login
-```
+    ```shell
+    gcloud init # Set $PROJECT as the default project
+    gcloud auth login
+    gcloud auth application-default login
+    ```
 
-### 0.3 kubectl
+3. Set up **`kubectl`**:
 
-You can install `kubectl` by running:
+    You can install `kubectl` by running:
 
-```shell
-gcloud components install kubectl
-```
+    ```shell
+    gcloud components install kubectl
+    ```
 
-### 0.4 Kubernetes clusters on GCP
+## 1. Create Kubernetes Engine Clusters
```

Comment:
Because you are changing this to GKE instead of k8s clusters on GCP, I think it's fine to make the command list more concise. But I really think we need to say something about k8s-on-GCE. Reference it here and have it somewhere. Anyway, it's up to you.

Reply:
I think I'll separate this to 3 sections:

1. Use existing Kubernetes Engine clusters
2. Create Kubernetes Engine clusters
3. Use existing non-GKE clusters on GCP

the last item would show how to merge contexts into a single kubeconfig file, while the instructions for GKE users will be simplified a great deal.